OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -13,18 +31,18 @@ $ docker network create -d bridge WCPortalNET
 ```
 
 ### B. WebCenter Content Server (Optional)
-You need to have a running WebCenter Content Server on any machine. The Content server connection details are required for configuring WebCenter Portal during configuration process.
-
-**Note :** This step is required only when the WebCenter Portal uses Content Server as content repository.
+If you want to integrate Oracle WebCenter Portal with Oracle WebCenter Content Server, you need to have a running Oracle WebCenter Content Server on any machine. But if you want to run  [Oracle WebCenter Content Server Container](../../../OracleWebCenterContent/dockerfiles/README.md) on the same machine as Oracle WebCenter Portal Container,
+then it should be running in same network as Oracle WebCenter Portal Container (use above created `WCPortalNET` network)
 
 ##  Running Oracle WebCenter Portal Docker Container
 To run the Oracle WebCenter Portal Docker container, you need to create:
 * Container to manage the Admin Server.
 * Container to manage the Managed Server.
```

Comment:
A container doesn't manage a managed server (the admin server does), so perhaps, "at least one managed server container".  If you support multiple managed servers, you could say "One or more managed server containers". If you only support one managed server, then just "a single managed server container"